### ISSUE TITLE
Crash on textarea custom component

### ISSUE THREAD

**Issue by hoseyjoe** [haxeui/haxeui-raylib#8]
The crash occurs on a custom component Text that extends the textarea. Text doesnt do anything but assign the below style.  Removing that and I believe it wont crash.
```
style="height:auto;min-height:26px;max-height:85px;width:100%;"
```

Also the crash doesn't occur when I add width="100%" to the component like in mainView
```
<text text="rief" "></text>
to
<text text="rief" width="100%"></text>
```

Crash only seems to occur when inside scrollview? Change main-view.xml to the below and it doesnt crash
```
<vbox width="100%" height="100%" style="padding: 5px;">
    <text text="rief"></text>
</vbox>
```

[ScrollBarCrash_original.zip](https://github.com/haxeui/haxeui-raylib/files/7173329/ScrollBarCrash_original.zip)

**Comment by hoseyjoe** [comment]
TLDR

Change custom component from style width to property width:
```
<textarea text="" style="height:auto;min-height:26px;max-height:85px;width:100%;"> 

</textarea>
```
to
```
<textarea text="" style="height:auto;min-height:26px;max-height:85px;" width="100%">    

</textarea>
```

Fixes issue.  Not sure why style causes crash but property change does not
